app/services/resolver_service.rb

```diff
@@ -0,0 +1,21 @@
+class ResolverService
+  def initialize(object, user)
+    @calling_object = object
+    @user = user
+  end
+
+  def process
+    mark_complete
+    # TODO: implement evidence_check create for applications
+    # TODO: implement payment creation for applications and evidence_checks
```

Comment:
End of the day, my brain is fried. Is this a reminder (the TODOs) for a future reference or for something that was meant to be added to this PR but was omitted ?